packages/fxa-settings/src/pages/Signup/container.tsx

```diff
@@ -73,14 +75,20 @@ const SignupContainer = ({
   const location = useLocation() as ReturnType<typeof useLocation> & {
     state: LocationState;
   };
-  const { emailStatusChecked } = location.state || {};
 
   const { queryParamModel, validationError } =
     useValidatedQueryParams(SignupQueryParams);
+  // emailStatusChecked is passed as a query param when coming from Backbone
+  // email-first, but location state when coming from React email-first.
+  // We can remove the query param bits once we're confidently at 100% roll out
+  // for React.
+  // emailStatusChecked can also be passed from React Signin when users hit /signin
+  // with an email query param that we already determined doesn't exist.
+  const emailStatusChecked =
+    queryParamModel.emailStatusChecked || location.state?.emailStatusChecked;
+  const email = queryParamModel.email || location.state?.email;
+  const shouldUseReactEmailFirst = useCheckReactEmailFirst();
 
-  // Since we may perform an async call on initial render that can affect what is rendered,
-  // return a spinner on first render.
```

Comment:
I removed this because very few users (only those hitting signup directly with an `email` query param, which I think Sync/RPs may do in some cases?) should have to wait for this call to complete, and if the account doesn't exist no rerender happens, but if the account does exist then those users will be redirected to `/signin`. You can test this by going to `/signin?email=test@gmail.com` assuming you've got an account created with that email. It does mean users in this bucket will see signup flashed for a split second (it's not even noticable locally).

The loading spinner on first render was less noticeable prior to React email-first but it makes the UI feel snappier to remove it now. I see looking now could do a `useState` for `emailStatusChecked`, but it sort of feels like overkill.